src/scripts/get-youtube-count.py

```diff
@@ -0,0 +1,71 @@
+# Requires file named client_secret.json in same directory, containing
+# OAuth 2.0 Client ID with youtube.readonly scope in YouTube Data API v3
+# https://console.developers.google.com/
+#
+# Required modules:
+# python -m pip install
+# mysql
+# mysql-connector
+# python-dotenv
+# google-api-python-client
+# google-auth-oauthlib
+
+import os
+import json
+import mysql.connector
+
+import google_auth_oauthlib.flow
+import googleapiclient.discovery
+import googleapiclient.errors
+from dotenv import load_dotenv
+
+load_dotenv()
+
+scopes = ["https://www.googleapis.com/auth/youtube.readonly"]
+
+db = mysql.connector.connect(
+    host=os.getenv("DB_HOST"),
+    user=os.getenv("DB_USER"),
+    password=os.getenv("DB_PASS"),
+    database=os.getenv("DB_KMQ_SETTINGS_TABLE_NAME")
+)
+
+cursor = db.cursor()
+
+cursor.execute("SELECT link FROM available_songs")
```

Comment:
`available_songs` needs a `last_updated` field, and this query should only attempt to update songs that are `x` days old